build.sbt

```diff
@@ -1,27 +1,62 @@
+addCommandAlias("check", "; scalafmtSbtCheck; scalafmtCheckAll; compile:scalafix --check ; test:scalafix --check")
+addCommandAlias("fix", "; scalafmtSbt; scalafmtAll; compile:scalafix ; test:scalafix")
+
 ThisBuild / organization := "com.avast"
 ThisBuild / homepage := Some(url("https://github.com/avast/scala-server-toolkit"))
 ThisBuild / description := "Functional programming toolkit for building server applications in Scala."
 ThisBuild / licenses := Seq("MIT" -> url("https://raw.githubusercontent.com/avast/scala-server-toolkit/master/LICENSE"))
 ThisBuild / developers := List(Developer("jakubjanecek", "Jakub Janecek", "janecek@avast.com", url("https://www.avast.com")))
-ThisBuild / scmInfo := Some(
-  ScmInfo(url("https://github.com/avast/scala-server-toolkit"), "scm:git:git@github.com:avast/scala-server-toolkit.git")
-)
 
 ThisBuild / scalaVersion := "2.13.0"
-ThisBuild / scalacOptions := ScalacOptions.default
 
 ThisBuild / turbo := true
 
 lazy val commonSettings = Seq(
   libraryDependencies ++= Seq(
     compilerPlugin(Dependencies.kindProjector),
+    compilerPlugin(Dependencies.betterMonadicFor),
+    compilerPlugin(Dependencies.silencer),
+    compilerPlugin(scalafixSemanticdb), // for scalafix
+    Dependencies.silencerLib,
     Dependencies.catsEffect,
-    Dependencies.scalaTest
+    Dependencies.scalaTest % Test
```

Comment:
I don't know which version you base this on but in newer version of `master` ScalaTest is already marked as `Test` scope so this shouldn't be needed.

Reply:
I've always (mostly?) seen people declare `% Test` explicitly, so I'm just following a convention

Reply:
I wouldn't call it a convention, I haven't seen it that much. Anyway, the difference between our approaches is very small. I am not against it, just I think you should now do whatever is in `master` and we can do another PR to update this style, ok?

Reply:
I would also vote to put `% Test` explicitly.